source/language/delays.rst

```diff
@@ -83,7 +83,7 @@ whatever their actual durations may be, we can do the following:
 
        barrier q;
        cx q[0], q[1];
-       u q[2];
+       U(pi/4, 0, pi/2) q[2];
```

Comment:
See also line 102 below--uses gate `u`, which is not defined. Should fill in angles--may need to update the image as well.